maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml

```diff
@@ -14,17 +44,19 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+#if ( ${javaCompilerVersion} == $null )
+#compilerProperties( "1.7" )
+#else
+#compilerProperties( ${javaCompilerVersion} )
+#end
   </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
-      <scope>test</scope>
-    </dependency>
+#if ( ${junitVersion} == $null )
+#junit( "4.12" )
```

Comment:
latest version is 4.13.1 ?
https://mvnrepository.com/artifact/junit/junit/4.13.1

Reply:
I've updated the version accordingly.